### PR TITLE
chore: add office hour blog file for 061023

### PR DESCRIPTION
**Pull request by almadigabor**

### CODE REVIEW

File: blog/2023-10-06-OfficeHours.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## General updates / information
+
+### DevSecOps / System-Team
+
+- Improvements on the Tractus-X [Release Guidelines Checks](https://eclipse-tractusx.github.io/sig-release/) page.
+  - Repositories are grouped by type (product, special interest groups, supporting).
+  - Repositories are now sorted in alphabetical order.
+  - The team is aware of some issues (like base image detection) and are investigating a solution.
+  - Please give us feedback if you find any issues or have some ideas on how to improve!
+- A [Release Planning board](https://github.com/orgs/eclipse-tractusx/projects/26/views/9)is available now on GitHub.
```
Comment (FaGru3n) [inline blog/2023-10-06-OfficeHours.md:17]: just a little blank
`A [Release Planning board](https://github.com/orgs/eclipse-tractusx/projects/26/views/9) is`

**Review by FaGru3n — CHANGES_REQUESTED** [review]
little typo

File: blog/2023-10-06-OfficeHours.md (hunk 1 above)
Comment (almadigabor) [reply blog/2023-10-06-OfficeHours.md]: thanks for the find, fixed it.

**Review by FaGru3n — APPROVED** [review]
LGTM

**Pull request merged**